contracts/storage/EternalStorage.sol

```diff
@@ -232,9 +195,16 @@ contract EternalStorage {
         stringArrayStorage[_key][_index] = _value;
     }
 
-        /////////////////////////////
-        /// Public getters functions
-        /////////////////////////////
+    /// Public getters functions
+    ////////////////////
+    /// @notice Get function use to get the value of the singleton state variables
+    /// Ex1- string public version = "0.0.1";
+    /// string _version = getString(keccak256(abi.encodePacked("version"));
+    /// Ex2 - assert(temp1 == temp2); replace to
+    /// assert(getUint(keccak256(abi.encodePacked(temp1)) == getUint(keccak256(abi.encodePacked(temp2));
+    /// Ex3 - mapping(string => SymbolDetails) registeredSymbols; where SymbolDetails is the structure having different type of values as
+    /// {uint256 date, string name, address owner} etc.
+    /// string _name = getString(keccak256(abi.encodePacked("registeredSymbols_name", "TOKEN"));
 
     function getUintValues(bytes32 _variable) public view returns(uint256) {
```

Comment:
Can you please rename functions like `getUintValues` to `getUintValue` as they are returning only 1 value.
Also, update the code comment above to show the new name.

Reply:
Done.